tensorboard/defs/web.bzl

```diff
@@ -100,7 +106,7 @@ def _tf_web_library(ctx):
       new_typings.append(src)
       new_typings_paths.append(entry[0])
       new_typings_execroot.inputs.append(entry)
-      ts_inputs += [src]
+      ts_inputs = depset([src], transitive=[ts_inputs])
```

Comment:
stupid question: I see two patterns of concatenation:
```py
# first
a = depset(transitive=[b, a])
# second
a = depset([b], transitive=[a])
```

Are they functionally different? If not, what is the reason to prefer one over the other?

Reply:
Yes, they are different. The elements of the `transitive` list must be
depsets, whereas the elements of `items` (the first positional
parameter) must be files. If you use the wrong one, you get an error.

Reply:
(Good question, by the way.)